templates/commands/render/render.go

```diff
@@ -39,10 +39,12 @@ import (
 	"github.com/abcxyz/abc/templates/common"
 	"github.com/abcxyz/abc/templates/common/specutil"
 	"github.com/abcxyz/abc/templates/common/templatesource"
+	"github.com/abcxyz/abc/templates/model"
 	spec "github.com/abcxyz/abc/templates/model/spec/v1beta2"
 	"github.com/abcxyz/pkg/cli"
 	"github.com/abcxyz/pkg/logging"
 	"github.com/abcxyz/pkg/sets"
+	ign "github.com/monochromegane/go-gitignore"
```

Comment:
I see this is used internally, but still not sure if it is ok to use this library here, if not, should we implement our own in pkg?

Reply:
I'm leaning toward not using it. It looks unmaintained and kinda sketchy, and there's not much actual code thereI think we'd be better writing these few lines of code ourselves.

Reply:
Sure, let me hold this PR until we implemented our own.

Reply:
Switched to use filepath glob.